<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceCfg_EditFonts</class>
 <widget class="QWidget" name="InterfaceCfg_EditFonts">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Font</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="listFonts"/>
   </item>
   <item row="1" column="1">
    <widget class="QListWidget" name="listSize"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lblSample">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>The Quick Brown Fox Jumps Over A Lazy Dog</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
